src/codec/meta/cid.rs

```diff
@@ -103,3 +102,9 @@ impl TryFrom<&str> for Cid {
         Ok(Cid::from(cid_bytes))
     }
 }
+
+impl Default for Cid {
+    fn default() -> Self {
+        Self([0u8; CID_LENGTH])
+    }
+}
```

Comment:
I think this generally falls under the same as the other Default implementations. We generally only want to be able to construct values that have meaning especially for any values that have very specific meanings.